### ISSUE TITLE
Pretrained model for day to night conversion

### ISSUE THREAD

**Issue by ankur-chr** [sauradip/night_image_semantic_segmentation#14]
I am trying to use the pretrained model for day-to-night conversion of images.
Is the pre-trained model using drop-out in the Generator? (The testing approach for cyclegan pytorch implementation asks for this flag i.e. whether the generator had drop-out)

**Comment by ankur-chr** [comment]
I am facing various issues just converting the day images to night images using the pre-trained model. Could you please help share some steps that can be used. (I tried using the CycleGAN from the github link provided, and also tried its Pytorch Colab notebook)

**Comment by ankur-chr** [comment]
Seems to work with the --no-dropout option while testing.

**Comment by ankur-chr** [comment]
Fixed
